--- a/ebtables-legacy.8.in
+++ b/ebtables-legacy.8.in
@@ -1,4 +1,4 @@
-.TH EBTABLES 8  "@PACKAGE_DATE@"
+.TH EBTABLES-LEGACY 8 "@PACKAGE_DATE@"
 .\"
 .\" Man page written by Bart De Schuymer <bdschuym@pandora.be>
 .\" It is based on the iptables man page.
@@ -24,7 +24,7 @@
 .\"     
 .\"
 .SH NAME
-ebtables-legacy (@PACKAGE_VERSION@) \- Ethernet bridge frame table administration (legacy)
+ebtables-legacy \- Ethernet bridge frame table administration (legacy)
 .SH SYNOPSIS
 .BR "ebtables " [ -t " table ] " - [ ACDI "] chain rule specification [match extensions] [watcher extensions] target"
 .br
